Index: shim-15+1533136590.3beb971/debian/rules
===================================================================
--- shim-15+1533136590.3beb971.orig/debian/rules
+++ shim-15+1533136590.3beb971/debian/rules
@@ -6,14 +6,9 @@ include /usr/share/dpkg/architecture.mk
 # dpkg-vendor --derives-from, because only Canonical-generated binaries will
 # be signed with this key; so if you are building your own shim binary you
 # should be building the other binaries also.
-ifeq ($(shell dpkg-vendor --is ubuntu && echo yes),yes)
-	cert=debian/canonical-uefi-ca.der
-	distributor=ubuntu
+cert=debian/cbld-uefi-ca.der
+distributor=cbld
 COMMON_OPTIONS ?= ENABLE_SHIM_CERT=1 ENABLE_SBSIGN=1
-else
-	cert=debian/debian-uefi-ca.der
-	distributor=debian
-endif
 
 DBX_LIST = dbx.esl
 DBX_HASHES = debian/$(distributor)-dbx.hashes
